diagram.md

```diff
@@ -91,7 +91,7 @@ The `Info Element` contains vital information for identifying the whole `Segment
 ```
 Figure: Representation of a `Info Element` and its `Child Elements`.
 
-The `Tracks Elements` tells us the technical details of what is in each track. For instance, is it a video, audio or subtitle track? What resolution is the video? What sample rate is the audio? The `Tracks Elements` can store the name, number, unique identifier, language, and type (audio, video, subtitles, etc) of each track. The `Tracks Element` also identifies what codec to use to decode the track and has the codec's private data for the track.
+The `Tracks Elements` defines the technical details for each track. The `Tracks Elements` can store the name, number, unique identifier, language, and type (audio, video, subtitles, etc) of each track. The `Tracks Element` can store information about the resolution of a video track or sample rate of an audio track. The `Tracks Element` also identifies what codec to use to decode the track and contains the codec's private data for the track.
```

Comment:
> The `Tracks Elements` defines…

`Elements` is plural; hence "define" and not "defines", or even better, make `Tracks Elements` singular (= `Tracks Element`, there's usually only one instance in a Document). If you make it singular, then all uses in this paragraph should be changed along with the corresponding verbs.